compiler/src/dotty/tools/dotc/typer/RefChecks.scala

```diff
@@ -622,15 +625,12 @@ object RefChecks {
       if (member.isAnyOverride && !(clazz.thisType.baseClasses exists (hasMatchingSym(_, member)))) {
         // for (bc <- clazz.info.baseClasses.tail) Console.println("" + bc + " has " + bc.info.decl(member.name) + ":" + bc.info.decl(member.name).tpe);//DEBUG
 
-        val nonMatching = clazz.info.member(member.name).altsWith(alt => alt.owner != clazz && !alt.is(Final))
-        def issueError(suffix: String) =
-          ctx.error(i"$member overrides nothing$suffix", member.pos)
+        val nonMatching = clazz.info.member(member.name).altsWith(alt => alt.owner != clazz)
```

Comment:
Just looking at the compressed diff here - so I can't see the reason to cut `!alt.is(Final)`